From: Richie (Hermes Enterprise)
Date: 2026-06-08
Subject: Add forceSecureDecoder field to ManifestConfig struct

Adds a boolean field to track the force_secure_decoder setting from
the addon's manifest_config property.

--- src/CompKodiProps.h
+++ src/CompKodiProps.h
@@ -95,6 +95,8 @@
   // If true will ignore the default KID provided from initialization segments
   // and so will be used from the manifest one
   bool ignoreMediaDefaultKid{false};
+  // Force the use of secure decoder (force_secure_decoder config from addon manifest_config)
+  bool forceSecureDecoder{false};
 };

 struct DrmCfg
